go/exorelay/glide.yaml

```diff
@@ -1,4 +1,7 @@
 package: github.com/Originate/exocom/go/exorelay
 import:
 - package: github.com/DATA-DOG/godog
-  version: v0.7.0
+  version: v0.7.2
+- package: golang.org/x/net
```

Comment:
This got added the the glide.lock file automatically since it realized it was a dependency. Making it more explicit here